protocol/message.go

```diff
@@ -25,7 +25,9 @@ type ErrorResponse struct {
 }
 
 type RegistrationResponse struct {
-	TempraryBinding string `json:"tb"`
+	RegEpoch uint64 `json:"reg_epoch"`
+	AP       string `json:"ap"`
+	TB       string `json:"tb"`
```

Comment:
I think the response should also contain the STR for ``RegEpoch`` since the client needs it to verify the TB signature as well as to validate the AP's root hash. If the STR is included do we still need ``RegEpoch``, too?